dissect/target/tools/shell.py

```diff
@@ -490,10 +490,18 @@ def do_info(self, line: str) -> Optional[bool]:
     @arg("-l", action="store_true")
     @arg("-a", "--all", action="store_true")  # ignored but included for proper argument parsing
     @arg("-h", "--human-readable", action="store_true")
-    def cmd_ls(self, args: argparse.Namespace, stdout) -> Optional[bool]:
+    @arg("-R", action="store_true", dest="recursive", help="Recursively list subdirectories encountered.")
+    @arg("-c", action="store_true", dest="use_ctime", help="Show time when file status was last changed.")
+    @arg("-u", action="store_true", dest="use_atime", help="Show time of last access.")
```

Comment:
```suggestion
    @arg("-R", "--recursive", action="store_true", help="recursively list subdirectories encountered")
    @arg("-c", action="store_true", dest="use_ctime", help="show time when file status was last changed")
    @arg("-u", action="store_true", dest="use_atime", help="show time of last access")
```
The lowercasing is mostly for consistency with an average bash shell.